stepactions/eaas-get-ephemeral-cluster-credentials/0.1/eaas-get-ephemeral-cluster-credentials.yaml

```diff
@@ -56,3 +64,18 @@ spec:
     "${OC[@]}" get secret $SECRET -o go-template --template="{{.data.kubeconfig|base64decode}}" > $CLUSTER_KUBECONFIG
     echo "Wrote kubeconfig to $CLUSTER_KUBECONFIG"
     echo -n $(basename $CLUSTER_KUBECONFIG) > $(step.results.kubeconfig.path)
+
+    # Get username and password
+    ADMIN_PASSWORD_SECRET=$("${OC[@]}" get cti $CLUSTER_NAME -o=jsonpath='{.status.adminPassword.name}')
+    echo "Found admin password secret: $ADMIN_PASSWORD_SECRET"
+    USERNAME=$("${OC[@]}" get secret $ADMIN_PASSWORD_SECRET -o jsonpath='{.data.username}' | base64 -d)
+    "${OC[@]}" get secret $ADMIN_PASSWORD_SECRET -o jsonpath='{.data.password}' | base64 -d > $CLUSTER_PASSWORD
```

Comment:
Nit: Go templates make it possible to do the b64 decoding in one command (see `base64decode` above). It'd be nice to use one approach in the same script for consistency.

Reply:
@amisstea Makes sense. I updated it